packs/tests/actions/test_installed_pack_version.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2019 Extreme Networks, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from st2common.constants.pack import PACK_VERSION_SEPARATOR
+from st2common.runners.base_action import Action
+from st2common.util.pack import get_pack_metadata
+from st2common.util.pack_management import get_repo_url
+
+
+class TesetInstalledPackVersionAction(Action):
+    def run(self, installed_pack):
+        """
+        :param installed_pack: Installed pack name with version
+        :type: installed_pack: ``string``
+        """
+
+        if not installed_pack:
+            return False, False
+
+        pack_and_version = installed_pack.split(PACK_VERSION_SEPARATOR)
+        pack_name = pack_and_version[0]
+        pack_version = pack_and_version[1] if len(pack_and_version) > 1 else None
+
+        # Pack version is not specified. Get pack version from index.json file.
+        if not pack_version:
+            try:
+                _, pack_version = get_repo_url(pack_name, proxy_config=None)
```

Comment:
Hm, I think there is a possible race condition here.

Let's say user installs pack ``xml`` without specifying a version when a latest version in index and git repo is v3.0.0. After the installation, but before running this check, new version is published to the index.

In such scenario, this check would (incorrectly) fail. I know the time window when this can happen is relatively small, but still something to keep in mind.

To avoid such issues, we should use some other test packs for testing such functionality, ones which rarely change (we already have some test packs for testing other pack related functionality, ``stackstorm-test``, ``stackstorm-test2``, etc.).